prow/cmd/phaino/local.go

```diff
@@ -55,11 +55,16 @@ func scanln(ctx context.Context) (string, error) {
 }
 
 func readMount(ctx context.Context, mount coreapi.VolumeMount) (string, error) {
-	fmt.Fprintf(os.Stderr, "local %s path (%q mount): ", mount.MountPath, mount.Name)
+	defaultMount := "mounts/" + mount.Name
+
+	fmt.Fprintf(os.Stderr, "local %s path (%q mount, default %q): ", mount.MountPath, mount.Name, defaultMount)
```

Comment:
The rest of this file uses `[default]` to note what will happen if you just press enter.

AKA `local %s path (%q mount) [%q]`